README.md

```diff
@@ -103,3 +105,50 @@ spec:
 ```
 
 After a few minutes, operators will be upgraded automatically.
+
+### Test serverless-operator with Istio sidecar injection
+
+To install service mesh operator and enable istio sidecar injection, run `make install-mesh`
+
+```
+make install-mesh 
+```
+
+Then, create Knative Service with `sidecar.istio.io/inject: "true"` annotation:
```

Comment:
Add a remark that it needs to be inside one of the namespaces of the member roll.

Reply:
Done.